### DISCUSSION TITLE
cpprb now available in conda-forge

### DISCUSSION THREAD

**Discussion by jamartinh** [ymd-h/cpprb#36] [Show and tell]
cpp is now available in conda-forge channel so you can install cpprb as:

```bash
>> conda install cpprb -c conda-forge
```

or 

```bash
>> mamba install cpprb -c conda-forge
```

**Comment by jamartinh** [comment]
for contributors, recipe available at https://github.com/conda-forge/cpprb-feedstock

**Comment by jamartinh** [comment]
Hi, has anyone tested the conda installation of cpprb ? is it working well ?
When I installed it on rhel8 python3.12 I receive core dump, and has to return to pip.

**Reply by ymd-h** [reply]
@jamartinh 

I have no experience in conda at all (and probably I won't use it in future too),
however, I feel `core dump` might come from NumPy ABI incompatibility.

I suggest you investigate versions of NumPy where you build and run cpprb.

As far as I know, NumPy has changed its ABI twice recently.
(aka. v1.19.5 to v1.20 and v1.x to v2.x)

cf. #34 , #25, #3
